src/Sylius/Bundle/UiBundle/DependencyInjection/SyliusUiExtension.php

```diff
@@ -40,6 +40,7 @@ public function load(array $configs, ContainerBuilder $container): void
 
         $container->setParameter('sylius_ui.twig_ux.anonymous_component_template_prefixes', $config['twig_ux']['anonymous_component_template_prefixes'] ?? []);
         $container->setParameter('sylius_ui.twig_ux.live_component_tags', $config['twig_ux']['live_component_tags'] ?? []);
+        $container->setParameter('sylius_ui.twig_ux.component_default_template', $config['twig_ux']['component_default_template'] ?? null);
```

Comment:
```suggestion
        $container->setParameter('sylius_ui.twig_ux.component_default_template', $config['twig_ux']['component_default_template']);
```

Reply:
I had to leave this `null` - without it half of unit tests for UiBundle fails

Reply:
Add `->addDefaultsIfNotSet()` to the `twig_ux` node, so it'll actually resolve the default values set there, also we're missing tests in `ConfigurationTest` regarding this node.